sdk/src/main/java/com/mapbox/maps/MapboxMap.kt

```diff
@@ -1691,6 +1695,54 @@ class MapboxMap :
     isMapValid = false
   }
 
+  /**
+   * Set map projection for Mapbox map.
+   *
+   * Note: as projection has become part of the style-spec it is better to use [setProjection] instead.
```

Comment:
From the user perspective - why better? And why this method is still here if it's worse?

Also - I'd probably wrote full `[Style.setProjection]` to make comment clearer

Reply:
it's worse because this method is experimental while another one is not